project.godot

```diff
@@ -26,7 +26,8 @@ Packet="*res://scripts/multiplayer/packet.gd"
 Anticheat="*res://scripts/multiplayer/anticheat.gd"
 Settings="*res://scripts/settings.gd"
 Modules="*res://scripts/modules.gd"
-TauntModule="res://modules/taunt/taunt.gd"
+KeywordModule="*res://modules/keyword/keyword.gd"
+TauntModule="res://modules/keyword/taunt/taunt.gd"
 RarityModule="res://modules/rarity/rarity.gd"
```

Comment:
Don't expose `RarityModule` since it doesn't have any children? Maybe?

Reply:
Superseded by https://github.com/LunarTides/Hearthstone.gd/pull/4#discussion_r1525871216